src/components/Button/Button.jsx

```diff
@@ -217,17 +217,17 @@ const Button = forwardRef(
 
     const leftIconSize = useMemo(() => {
       if (typeof leftIcon !== "function") return;
-      return "24";
+      return "20";
```

Comment:
I see this number and a couple more being used in multiple components, why not add this to a constant?